lab_2_retrieval_w_bm25/main.py

```diff
@@ -33,7 +44,18 @@ def remove_stopwords(tokens: list[str], stopwords: list[str]) -> list[str] | Non
 
     In case of corrupt input arguments, None is returned.
     """
+    if not isinstance(tokens, list) or not isinstance(stopwords, list) \
+            or not all(isinstance(token, str) for token in tokens) \
+            or not all(isinstance(token, str) for token in stopwords):
+        return None
+
+    lst_without_stops = []
```

Comment:
may be words_without_sw? or something like that

Reply:
fixed